src/test/resources/features/AddProductToCart.feature

```diff
@@ -3,8 +3,45 @@ Feature: Add product to Shopping cart
 
   Background:
     Given user is on Home page
+    And user clicks on 'electronics' category from category bar
+    And user clicks on 'cell-phones' subcategory
 
   Scenario: Validate access to subcategory
-    When User clicks on 'electronics' category from category bar
-    And User clicks on 'cell-phones' subcategory
-    Then 'Cell phones' subcategory page is displayed
\ No newline at end of file
+    Then 'Cell phones' subcategory is displayed
```

Comment:
In my opinion this  should be moved to the Backgraoun

Reply:
That's my TC1 result.